### ISSUE TITLE
create function to change AED nml files as is possible for GLM nml file

### ISSUE THREAD

**Issue by eread-usgs** [USGS-R/glmtools#157]

**Comment by jread-usgs** [comment]
:+1:

**Comment by jpdoubek** [comment]
I'll plus one that

**Comment by jread-usgs** [comment]
we are going to be working on this now. Will document here.

**Comment by jread-usgs** [comment]
A couple of changes here. 
- the variable names are not unique, and one name can exist in several blocks
- the current parser condenses the string vectors (e.g., "FLOW,TEMP,SALT,OXY_oxy,SIL_rsi,NIT_amm,NIT_nit") so that seems to be another issue

**Comment by jread-usgs** [comment]
Current syntax:
reading in fabm.nml w/ `read_nml()`
```R
get_nml_value(aed_nml, 'aed_oxygen::Fsed_oxy')
[1] -50
get_nml_value(aed_nml, 'Fsed_oxy')
```
'Fsed_oxy' is in (at least) two blocks. The simple syntax returns the first variable name match and doesn't warn that there may be more than one match (probably not desired). The `::` syntax specifies {block name}::{arg name} to be more explicit

**Comment by jread-usgs** [comment]
Update. the function now warns if more than one match was found:
```R
get_nml_value(aed_nml, 'Fsed_oxy')
[1] -3
Warning message:
In get_nml_value(aed_nml, "Fsed_oxy") :
  Fsed_oxy returned 2 matches, returning the first

```

**Comment by jread-usgs** [comment]
Update. A better warning message:
```R
get_nml_value(aed_nml, 'Fsed_oxy')
[1] -3
Warning message:
In get_nml_value(aed_nml, "Fsed_oxy") :
  Fsed_oxy found in aed_sed_constant & aed_oxygen, returning the first. Try aed_sed_constant::Fsed_oxy for explicit match
```

**Comment by lawinslow** [comment]
Are you doing set_nml_value as well?

**Comment by jread-usgs** [comment]
yep, same deal for that. Working on it now

**Comment by jread-usgs** [comment]
added some tests for set too, but the syntax is the same:
```R
nml = set_nml(aed_nml, 'aed_sed_constant::Fsed_oxy', arg_val = 1.5)
```
or 
```R
nml = set_nml(aed_nml, 'Fsed_oxy', arg_val = 1.5)
```
which will warn if there is more than one match to the arg_name

**Comment by jread-usgs** [comment]
Anyone up for trying this out?

**Comment by jread-usgs** [comment]
fixed with #177
